plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -46,6 +47,11 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         "Enable aggregation pushdown",
                         jdbcMetadataConfig.isAggregationPushdownEnabled(),
                         false))
+                .add(booleanProperty(
+                        CASE_INSENSTIVE_PREDICATE_PUSHDOWN_ENABLED,
+                        "Enable case insensitive predicate pushdown",
+                        jdbcMetadataConfig.isCaseInsensitivePredicatePushdownEnabled(),
+                        false))
```

Comment:
Why do we need session property? May be hidden?

Reply:
That makes tests easier. Also I see a case where user would like to use different values depending on query (not sure how likely it is).